### DISCUSSION TITLE
How to build this for Ubuntu 20.04?

### DISCUSSION THREAD

**Discussion by mavavilj** [Ifsttar/I-Simpa#289] [General]
How to build this for Ubuntu 20.04?

Or is using the Windows version preferred?

https://github.com/Ifsttar/I-Simpa/blob/master/Docs/Building.md

These are from 2017, not sure if they will still work.

**Comment by nicolas-f** [comment]
Currently appveyor is building I-Simpa automatically on windows. The build instructions for the virtual ubuntu machine are here:

https://github.com/Ifsttar/I-Simpa/blob/master/appveyor.yml

Since the building guide only the library versions have been upgraded:

boost_1_73_0
python38
wxWidgets-3.1.4
swigwin-3.0.11

**Reply by mavavilj** [reply]
Can you clarify? What virtual ubuntu machine? I'm using native Ubuntu.
Also, there's no mention of Ubuntu or Linux there:
https://github.com/Ifsttar/I-Simpa/blob/master/appveyor.yml

**Reply by mavavilj** [reply]
What about this:
https://github.com/Ifsttar/I-Simpa/blob/master/Docs/Building.md#on-gnulinux-gcc-or-clang
?

**Reply by nicolas-f** [reply]
I've made the updates, its still in the PR
https://github.com/Ifsttar/I-Simpa/pull/290/files

**Reply by mavavilj** [reply]
I still don't understand what I need to do for Ubuntu.

**Reply by mavavilj** [reply]
I also think that the manual build here:

https://github.com/Ifsttar/I-Simpa/blob/master/Docs/Building.md#on-gnulinux-gcc-or-clang

Contains too many steps, i.e. I find that there's a risk that it will not work out.

Are all the dependencies okay to install with apt for example?

**Reply by mavavilj** [reply]
This says bionic.

https://github.com/Ifsttar/I-Simpa/blob/master/.travis.yml

So does this work for Ubuntu 18.04?
